test/topo_flavor/etcd.py

```diff
@@ -1,21 +1,23 @@
 #!/usr/bin/env python
 
-# Copyright 2014, Google Inc. All rights reserved.
+# Copyright 2014 Google Inc. All rights reserved.
 # Use of this source code is governed by a BSD-style license that can
 # be found in the LICENSE file.
 
+"""Etcd specific configuration."""
+
 import os
 import shutil
 
 import server
 
 
-class EtcdCluster:
+class EtcdCluster(object):
   """Sets up a global or cell-local etcd cluster."""
 
   def __init__(self, name):
-    import environment
-    import utils
+    import environment  # pylint: disable=g-import-not-at-top
```

Comment:
This doesn't look right, please remove all pylint comments in this file related to imports, they should be on the top...